include/ignition/math/Helpers.hh

```diff
@@ -740,6 +786,52 @@ namespace ignition
       return {s.count(), ns.count()};
     }
 
+    // TODO(anyone): Replace this with std::chrono::days.
+    /// This will exist in C++-20
+    typedef std::chrono::duration<uint64_t, std::ratio<86400>> days;
+
+    /// \brief break down durations
+    /// \param[in] d Duration to breaw down
+    /// \return A tuple based on the durations specified
+    template<class...Durations, class DurationIn>
+    std::tuple<Durations...> break_down_durations(DurationIn d) {
```

Comment:
Style: 

```suggestion
    std::tuple<Durations...> breakDownDurations(DurationIn d) {
```

Also, it would be nice to explain a bit more what this function does on the `brief` above.